### ISSUE TITLE
Custom screen image not working on 21:9 aspect ratio

### ISSUE THREAD

**Issue by Aerograch** [WesternGamer/CustomLoadingBackgrounds#5]
Hello! I have been using your plugin for about 1 year, and it served me well. But, recently i have switch monitor from 1920x1080(16:9) to 3440x1440(21:9), and since then plugin fail to show any custom background image. Now it looks smth like this:
![image](https://github.com/WesternGamer/CustomLoadingBackgrounds/assets/65912076/bcc2b940-f625-4e55-aad4-96974cf48689)
How to fix it?

**Comment by WesternGamer** [comment]
Does the issue cause custom background images to not show up at all? Also were you using custom backgrounds in the loading screen?

**Comment by Aerograch** [comment]
Yes, there were no custom background images at all. And, if i understood that correctly, yes, i was using 8 png's as custom background(well, i dropped them into "LoadingScreenBackgroundImages" folder as usual, and that did not caused them to appear as custom backgrounds)

**Comment by Aerograch** [comment]
Also, i have reloaded SE and my pc several times, just to be sure.

**Comment by Aerograch** [comment]
Update:
I aslo tried custom menu background - it worked, but aspect ratio is wrong. Original png is 3440x1440 but plugin compressed it to 2560x1440:
![image](https://github.com/WesternGamer/CustomLoadingBackgrounds/assets/65912076/715a8610-1431-4d10-a46e-97f5956a2bdb)
Original image:
![5fdn71rodp2c1](https://github.com/WesternGamer/CustomLoadingBackgrounds/assets/65912076/cc4c1a03-a025-4abc-a3a0-9ac17c068b8b)

**Comment by Codebreakerblue** [comment]
Same issue here. My monitor is 2560x1080, and custom menu images are squashed to fit a 16:9 aspect ratio (I think).
Screenshot:
![image](https://github.com/user-attachments/assets/2ff425b9-2ee0-4168-85ce-b63b9f30a946)
Original image:
![News_header_image_1920px_Space_Engineers_PS_4_dd486bc81b-3064314587-resized](https://github.com/user-attachments/assets/53ad627d-6308-4c54-9e89-e6c1180bbf5c)
